src/hotspot/share/jvmci/jvmci_globals.hpp

```diff
@@ -170,7 +170,19 @@ class fileStream;
           "Enables intrinsification of BigInteger.montgomeryMultiply()"))   \
                                                                             \
   NOT_COMPILER2(product(bool, UseMontgomerySquareIntrinsic, false, DIAGNOSTIC, \
-          "Enables intrinsification of BigInteger.montgomerySquare()"))
+          "Enables intrinsification of BigInteger.montgomerySquare()"))     \
+                                                                            \
+  NOT_COMPILER2(product(bool, EnableVectorSupport, false, EXPERIMENTAL,     \
+          "Enables VectorSupport intrinsics"))                              \
+                                                                            \
+  NOT_COMPILER2(product(bool, EnableVectorReboxing, false, EXPERIMENTAL,    \
+          "Enables reboxing of vectors"))                                   \
+                                                                            \
+  NOT_COMPILER2(product(bool, EnableVectorAggressiveReboxing, false, EXPERIMENTAL, \
+          "Enables aggressive reboxing of vectors"))                        \
+                                                                            \
+  NOT_COMPILER2(product(bool, UseVectorStubs, false, EXPERIMENTAL,          \
+          "Use stubs for vector transcendental operations"))                \
```

Comment:
nit: remove the backslash?
```suggestion
          "Use stubs for vector transcendental operations"))
```

Reply:
I added the backslash for consistency with C2: https://github.com/openjdk/jdk/blob/master/src/hotspot/share/opto/c2_globals.hpp#L776-L778. It also makes future changes nicer, since appending more flags will not need to touch unrelated lines.

Reply:
Yes. Agree. Thanks.